packages/python-sdk/e2b/sandbox_sync/main.py

```diff
@@ -209,6 +213,7 @@ def set_timeout(  # type: ignore
         timeout: int,
         request_timeout: Optional[float] = None,
     ) -> None:
+        """Configure request timeout"""
```

Comment:
This does configure the *sandbox* timeout.
We might consider changing the method's name if the use is not apparent though.

Reply:
Got it, something like `set_sandbox_timeout` ? In the meanwhile i updated the docstring